setting up Miniconda...
activating virtual env...
current conda environment: basic310
activating ssh connection...
Agent pid 2414010
Identity added: /home/11864265/.ssh/id_ed25519 (11864265@fnwi-h0.science.uva.nl)
parsing input arguments...
Branch: feature/retrain_grammar
fetching and pulling latest changes from remote dir...
Already up to date.
running perturbed masking...
Started to load full PCFG...
Finished loading PCFG...
Traceback (most recent call last):
  File "/home/11864265/Documents/thesis_code/reduce_grammar/generate_subset.py", line 267, in <module>
    subset_pcfg, subset_pcfg_pos = load_subset_pcfg(prod_productions_v2, args, top_k=k, save=args.save, load=args.load, lexical=args.lexical)
TypeError: load_subset_pcfg() got multiple values for argument 'top_k'
srun: error: fnwi-cn005: task 0: Exited with exit code 1
adding, committing, and pushing results to remote dir...
